app/assets/stylesheets/components/cards.scss

```diff
@@ -42,3 +42,25 @@
     padding: var(--su-4);
   }
 }
+
+.c-card {
```

Comment:
I honestly still trying to figure it out. I think like it's not the best approach to add such a component but this component is generally pretty weird/complicated from a design perspective... Sooo.. This is kinda todo for me...